doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -48,7 +48,11 @@ This specification is licensed under the
 
 ## Conformance
 
-All the interfaces described by this specification are mandatory and must be implemented in order to be compliant with. 
+This specification describes the "full" compliance level.
+
+However, next iterations of this specification could define optionality
+in some parts and the compliance level (or roles/profiles)
+corresponding to them.
```

Comment:
Lack of answers make me thing that we should keep discussing on this. Thus, NTC for this PR with regard this (eventual future modifications would be done in a next PR).